app/models/manufacturer.rb

```diff
@@ -2,4 +2,8 @@ class Manufacturer < ApplicationRecord
   has_many :materials
 
   validates :name, presence: true
+
+  def self.names
+    all.map{|material| [material.name, material.id]}
```

Comment:
`pluck(:name, id).to_a` will be more efficient I think as it will avoid creating full manufacturer objects in memory for only two attributes.